### DISCUSSION TITLE
EW-7811Un working on Mac Pro 2009 Monterey 12.6.8 (OCLP)

### DISCUSSION THREAD

**Discussion by Edgecrusherr160** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#310] [General]
I just wanted to report this is working with an Edimax EW-7811Un on a Mac Pro 2009, under macOS 12.6.8 Monterey, via OpenCore Legacy Patcher v0.6.8. 

The performance is pretty slow, but I think that's due to where I have a computer located. It's mostly just being used as a back up option, in case the very loose ethernet cable comes in plugged. 

I'm pretty surprised overall, I thought this was going to be a failure, due to the statement on the first page about it being for Hackintoshes, but after looking into the project, it looks like it was a pretty good shot on a real Mac.

**Comment by chris1111** [comment]
Thanks @Edgecrusherr160 
I will surely edit the page by adding OCLP support because OpenCore is used
